protocols/relay/src/v2.rs

```diff
@@ -42,48 +42,53 @@ pub mod client {
 
         #[deprecated(
             since = "0.15.0",
-            note = "Use libp2p_relay::client::Transport instead."
+            note = "ClientTransport has been renamed to Transport
+            and is now private as it shouldn't be required by end users."
         )]
-        pub type ClientTransport = crate::client::Transport;
+        pub type ClientTransport = crate::priv_client::transport::Transport;
 
         #[deprecated(
             since = "0.15.0",
-            note = "Use libp2p_relay::client::transport::Listener instead."
+            note = "RelayListener has been renamed to Listener
+            and is now private as it shouldn't be required by end users."
```

Comment:
```suggestion
            note = "RelayListener will become crate-private in the future as it shouldn't be required by end users."
```

I don't think we need to tell them about the rename :)

Reply:
yeah makes sense, updated!